src/metabase/lib/drill_thru.cljc

```diff
@@ -71,28 +102,19 @@
   ([query        :- ::lib.schema/query
     stage-number :- :int
     context      :- ::lib.schema.drill-thru/context]
-   (let [context (icky-hack-add-source-uuid-to-aggregation-column-metadata query stage-number context)]
-     (try
-       (into []
-             (keep #(% query stage-number context))
-             ;; TODO: Missing drills: automatic insights, format.
-             [lib.drill-thru.distribution/distribution-drill
-              lib.drill-thru.column-filter/column-filter-drill
-              lib.drill-thru.foreign-key/foreign-key-drill
-              lib.drill-thru.object-details/object-detail-drill
-              lib.drill-thru.pivot/pivot-drill
-              lib.drill-thru.quick-filter/quick-filter-drill
-              lib.drill-thru.sort/sort-drill
-              lib.drill-thru.summarize-column/summarize-column-drill
-              lib.drill-thru.summarize-column-by-time/summarize-column-by-time-drill
-              lib.drill-thru.underlying-records/underlying-records-drill
-              lib.drill-thru.zoom-in-timeseries/zoom-in-timeseries-drill])
-       (catch #?(:clj Throwable :cljs :default) e
-         (throw (ex-info (str "Error getting available drill thrus for query: " (ex-message e))
-                         {:query        query
-                          :stage-number stage-number
-                          :context      context}
-                         e)))))))
+   (try
+     (into []
+           (mapcat (fn [context]
+                     (let [context (icky-hack-add-source-uuid-to-aggregation-column-metadata query stage-number context)]
```

Comment:
The above-mentioned changes include a better way of handling aggregations. Same decision.